pallets/schemas/src/lib.rs

```diff
@@ -88,7 +88,7 @@ pub mod pallet {
 	#[pallet::config]
 	pub trait Config: frame_system::Config {
 		/// The overarching event type.
-		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
```

Comment:
Was this also renamed in the new substrate version?

Reply:
Yes

Reply:
https://github.com/substrate-developer-hub/substrate-parachain-template/pull/133